utils/constants.js

```diff
@@ -62,4 +62,20 @@ export const FOOTER_SOCIALS = [
     icon: 'facebook',
     url: 'www.facebook.com',
   },
+]
+
+// for use with the BootstrapFooter component
+export const FOOTER_LINKS = [
+  {
+    name: 'The Team',
+    url: '/about-us',
+  },
+  {
+    name: 'Services',
+    url: '/services',
+  },
+  {
+    name: 'Contact',
+    url: '/contact',
+  },
```

Comment:
if we aren't actually using Footer2 in this app...then I don't think we need the constants set up for it. 

if an app wants to use any components that are not included in this prototype, then the understanding will be that new constants, etc. will have to be created for that use case.